### DISCUSSION TITLE
Inpainting Model Update

### DISCUSSION THREAD

**Discussion by chaowentao** [open-mmlab/mmediting#1220] [General]
Now，SOTA Inpainting Model is updated，such as(ProFill, HiFill, etc). Can you have the plan?

**Comment by innerlee** [comment]
Hi @chaowentao thanks for the inquiry. Currently we do not have plan in implementing inpainting models due to limited manpower capacity. Nevertheless, two notes are always true:

* Community contributions are all welcome! Even the pr might look in early stage, don't hesitate and we will help the author to polish.
* Join open-mmlab!
